components/User-feed-status.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Icon from './icon';
+
+const UserFeedStatus = (props) => <>
```

Comment:
Let's destructure `props` parameter here, as we do in other components:

Before: `(props)`

After: `({ isGone, isPrivate, isProtected, isRestricted, type })`